### DISCUSSION TITLE
ImportError: cannot import name 'Markup' from 'flask'

### DISCUSSION THREAD

**Discussion by glopglopoups** [pallets/flask#5293] [Q&A]
I encountered the following error when switching from flask to version 3.0:

`ImportError: cannot import name 'Markup' from 'flask'`

The evolution of flask impacte "flask_navigation" and "flaskext.markdown" in my case. It is possible that other modules related to flask can be impacted.

Do you have a corrective solution, either for the additional modules or for flask to adapt to its evolution?

**Comment by RonnyPfannschmidt** [comment]
as per the 3.xx those deprecated aliases got removed in #5223

**Reply by glopglopoups** [reply]
That’s what I understood. I wanted to point that out because many modules associated with Flask have not integrated this evolution.

**Reply by davidism** [reply]
Then you should open up issues or PRs with them, if they aren't already written.

**Reply by RonnyPfannschmidt** [reply]
Flask had deprecation in place for a while before,so it's not clear to me what the point/ask is
